src/Compiler/SyntaxTree/PrettyNaming.fs

```diff
@@ -474,33 +520,35 @@ let DecompileOpName opName =
             opName
 
 let DoesIdentifierNeedBackticks (name: string) : bool =
-    not (IsUnencodedOpName name) && 
-    not (IsIdentifierName name) && 
-    not (IsActivePatternName name)
+    not (IsUnencodedOpName name)
+    && not (IsIdentifierName name)
+    && not (IsActivePatternName name)
```

Comment:
@dsyme it looks like there are many places where operators got recently moved to the next line and the most of the compiler codebase used to have it on the previous line. What do you think about making it a preferred default in the style guide to have it on the previous line, as it used to be?

Reply:
If fantomas has this option I'd consider using it. However I cant see such an option here? https://github.com/fsprojects/fantomas/blob/master/docs/Documentation.md

Reply:
@nojaf What do you think about adding such an option?

Reply:
https://github.com/fsprojects/fantomas/blob/master/CONTRIBUTING.md#what-are-we-not-looking-for

This has been going on for some weeks now, so allow me to spell it out one last time:
The style is determined by the guide, which is discussed over at [fsharp/fs-langdesign](https://github.com/fsharp/fslang-design#style-guide).
Please don't ask for settings but convince your peers of your idea so we can have better defaults.

Reply:
Yes, I've started with the style guide question.